src/main/java/io/github/swagger2markup/Swagger2MarkupProperties.java

```diff
@@ -219,6 +222,34 @@ public URI getRequiredURI(String key) {
     }
 
     /**
+     * Return a list of Path property values associated with the given key, 
+     * or {@code defaultValue} if the key cannot be resolved.
+     * 
+     * @param key the property name to resolve
+     * @return The list of Path properties
+     * @throws IllegalStateException if the value cannot be mapped to an array of strings
+     */
+    public Optional<List<Path>> getPathList(String key) {
```

Comment:
I think we should not return an Optional here. We should return an empty list instead.